schema.sql

```diff
@@ -48,4 +48,6 @@ CREATE TABLE visits (
     vets_id INT ,
     FOREIGN KEY (vets_id) REFERENCES vets,
     date_of_visit DATE
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=the%20initial%20data.-,Run%20the%20following%20query%20to%20add%20an%20extra%20column%20to%20the%20owners%20table%3A,-%2D%2D%20Add%20an), Please add column `email` to owners table, I can see that you added the comment but you didn't add the column itself.